appcues/src/main/java/com/appcues/ui/extensions/DrawScopeEx.kt

```diff
@@ -11,39 +9,25 @@ import java.lang.Float.max
 /**
  * Defines a path to draw the X by drawing two lines crossing from edge to edge.
  */
-internal fun DrawScope.xShapePath(
-    color: Color,
-    pathSize: Dp,
-    strokeWidth: Dp,
-): Path {
+internal fun DrawScope.xShapePath(pathSize: Dp): Path {
     return Path()
         .apply {
-            val strokeWidthPx = strokeWidth.toPx()
             val pathSizePx = pathSize.toPx()
+            val sizeRect = size.toRect()
 
             // center the desired size within the current bounds
+            // find the delta to deflate the container rect in each direction
             val deltaX = max(size.width - pathSizePx, 0.0f) / 2
             val deltaY = max(size.height - pathSizePx, 0.0f) / 2
```

Comment:
the centering detail here becomes important, as now the button height and width don't necessarily have to match - can have any rounded rect shape. We sill want the `X` to be visually centered within and kept in a square bounds itself. Prior to this with just a "deflate" option (previous PR) - you could end up with the `X` getting stretched on one axis.